pkgs/tools/misc/journaldriver/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, fetchFromGitHub, rustPlatform, pkgconfig, openssl, systemd }:
+
+rustPlatform.buildRustPackage rec {
+  name        = "journaldriver";
+  version     = "1.0.0-rc2";
+  cargoSha256 = "04llhriwsrjqnkbjgd22nhci6zmhadclnd8r2bw5092gwdamf49k";
+
+  src = fetchFromGitHub {
+    owner  = "aprilabank";
+    repo   = "journaldriver";
+    rev    = "v${version}";
+    sha256 = "19vb6hk65cvf4401hjg52q8ihkhcykrx72llv7m08bwpz59rycs2";
+  };
+
+  # Disable tests in nixpkgs
+  doCheck = false;
+
+  buildInputs = [ pkgconfig openssl systemd.dev ];
```

Comment:
`buildInputs` should choose the dev outputs automatically, so I think it should also work with `s/systemd.dev/systemd`

Reply:
Also, `pkgconfig` should go in `nativeBuildInputs`